telescope/utils.py

```diff
@@ -387,10 +387,17 @@ async def fetch_bigquery(sql):  # pragma: nocover
 
     def job():
         bqclient = getattr(threadlocal, "bqclient", None)
+
         if bqclient is None:
-            # Reads credentials from env and connects.
-            bqclient = bigquery.Client()
+            if config.BIGQUERY_PROJECT_ID:
+                bqclient = bigquery.Client(project=config.BIGQUERY_PROJECT_ID)
+
+            else:
+                # Reads credentials from env and connects.
+                bqclient = bigquery.Client()
```

Comment:
I think that `bqclient = bigquery.Client(project=config.BIGQUERY_PROJECT_ID)` would be enough, because when looking at docs https://cloud.google.com/python/docs/reference/bigquery/latest/google.cloud.bigquery.client.Client you can see that `None` is the default value

Reply:
thanks! simplified this :+1: